files/sysdig-inline-scan.sh

```diff
@@ -14,7 +14,7 @@ SCAN_IMAGE=""
 VALIDATED_OPTIONS=""
 # Vuln scan option variable defaults
 DOCKERFILE=""
-TMP_PATH="/tmp/sysdig"
+TMP_PATH=${TMP_PATH:-"/tmp/sysdig-inline-scan"}
```

Comment:
Is this change somehow involved with the time profile changes this PR is about? Am I missing something?

Reply:
Ups, it went all into the same PR, but in fact, this and the change in line 295 are different, let me put them in a different PR.